internal/database/repository/repository.go

```diff
@@ -27,7 +27,7 @@ type Repository interface {
 	GetNodesPerPartition(ctx context.Context, partition string) ([]*dao.NodeDAOInfo, error)
 	UpsertPartitions(ctx context.Context, partitions []*dao.PartitionInfo) error
 	GetAllPartitions(ctx context.Context) ([]*dao.PartitionInfo, error)
-	UpsertQueues(ctx context.Context, queues []*dao.PartitionQueueDAOInfo) error
+	UpsertQueues(ctx context.Context, parentId *string, queues []*dao.PartitionQueueDAOInfo) error
```

Comment:
Are there any semantic differences if `parentId` is `null` opposed to `""`?

Reply:
I made it a pointer to a string as `queues.parent_id` is nullable in the db, and @sudiptob2's `getQueueId()` already returned a `*string`.

Reply:
The `UpsertQueues()` is now deprecated (we will remove it when this PR, which adds `AddQueues()` and the PR for `UpdateQueue()` lands). Iif a pointer to an empty string var is passed in a queue struct to `AddQueues()`, the insert should fail, as a valid UUID is required for that column. (The db column is literally of type `UUID`.)